README.md

```diff
@@ -1081,6 +1084,11 @@ Usage: `nerdctl volume rm [OPTIONS] VOLUME [VOLUME...]`
 
 ## Namespace management
 
+### :nerd_face: :blue_square: nerdctl namespace create
+Create a new namespace.
+
+Usage: `namespaces create NAMESPACE [<key>=<value]`
```

Comment:
`<key>=<value>` should be `--label`

Reply:
like  “nerdctl ns label namespace1  label  key1=value1,there is double lable ,” can you for an example like "nerdctl ns label namespace1 key1=value1" ?

Reply:
can you for an example use --label ? @AkihiroSuda

Reply:
```
nerdctl namespace create foo --label key1=val1 --label key2=val2

nerdctl namespace update foo --label key3=val3 --label key4=val4
```

Reply:
i use nerdctl namespace update foo  --label "key1 =v1 key2 =v2", cobra doesn't support repeat flag

Reply:
> cobra doesn't support repeat flag

Untrue. We have `nerdctl run -v vol1 -v vol2` .

Reply:
done,thank you